### ISSUE TITLE
Can I input a image that w is not equal to the h?

### ISSUE THREAD

**Issue by 66ling66** [CompVis/zigma#10]

**Comment by Hiccupwzy** [comment]
I think the image `w` can be not equal to `h` for that you can simply resize it into your preferred size. 
And I have a small question for you. Did you reproduce the sample result follow the `faceshq1024_0060000.pt`. I followed the instruction, but got corrupt pictures rather than the clean face.
 Thank you for your help.
![image](https://github.com/CompVis/zigma/assets/151977184/55354ab3-4634-417c-9dce-0bde5d919691)

**Comment by Hiccupwzy** [comment]
@66ling66

**Comment by AY-Liu** [comment]
> I change this line [decode](https://github.com/CompVis/zigma/blob/4f6ff22ea104927eea8e2a88e6149f987ac20805/sample_acc.py#L367) and divide by 0.18215. Then the raw picture can turn to raw colorful picture. But the generated picture still be corrupted. ![image](https://private-user-images.githubusercontent.com/151977184/331012701-70a97149-debf-4207-a137-52bc41d106ad.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NjczODQsIm5iZiI6MTcxNTg2NzA4NCwicGF0aCI6Ii8xNTE5NzcxODQvMzMxMDEyNzAxLTcwYTk3MTQ5LWRlYmYtNDIwNy1hMTM3LTUyYmM0MWQxMDZhZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxNlQxMzQ0NDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMTEwODYwOTZiN2YwODNhMTdmMzhlMGZjNTE5MTY3NjMyNTI2ZTNlZGRhYjIzNDBhMzE5MGU4ZmI0NzgyZjUzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.AHa5rERMYjKllWbgRar-iQafVVOrGTjfSEHhk3cDtZg)

sample problem. Do you solve it?

**Comment by Hiccupwzy** [comment]
Still not. But I retrained the facehq1024 using zigman8 config file. Honestly, the performance of my model is not good as that mentioned in the paper.

**Comment by AY-Liu** [comment]
> Still not. But I retrained the facehq1024 using zigman8 config file. Honestly, the performance of my model is not good as that mentioned in the paper.

Dude, can you tell me how to process facehq data into webdataset form? I'm not quite sure how that training set and test set tar of theirs is split and composed, I have ffhq in png and latent form now, what should I do next?

**Comment by dongzhuoyao** [comment]
If the input W and H is not equal. You can try:

1) Padding the images.
2) implement a space-fillling curve yourself, you can get some intuition from here: https://github.com/galtay/hilbertcurve
